docs/readthedocs/source/doc/Orca/QuickStart/orca-tf2keras-quickstart.md

```diff
@@ -118,6 +118,35 @@ est.shutdown()
 print(stats)
 ```
 
+### **Step5: Save and Load the Model**
+
+Orca TF2 Estimator supports 2 formats to save and load the entire model (**TensorFlow SavedModel and Keras H5 Format**). The recommended format is SavedModel, it is the default when you use `estimator.save()`.
```

Comment:
2 -> two
it is the default when you use `estimator.save()`. -> which is the default format when ...

Reply:
fixed.